.qlty/qlty.toml

```diff
@@ -91,10 +91,11 @@ name = "yamllint"
 
 # Disable linters which are likely to produce false positives
 # on our test fixtures directories
-[[ignore]]
+[[exclude]]
 plugins = ["osv-scanner", "trufflehog"]
 file_patterns = ["qlty-cli/tests/**"]
 
-[[ignore]]
-rules = ["markdownlint:MD024"]
-file_patterns = ["CHANGELOG.md"]
+# Temporarily comment this.
```

Comment:
Is this intended?

Reply:
Yes rules are no longer supported by the new exclude [syntax](https://github.com/qltysh/cloud/issues/5925) and this should be moved to [triage](https://github.com/qltysh/qlty/pull/2005) once that lands.